sources/packages/forms/src/form-definitions/sfaa2022-23.json

```diff
@@ -30290,7 +30288,128 @@
               "inputType": "radio",
               "fieldSet": false,
               "id": "e26wllo",
-              "defaultValue": ""
+              "defaultValue": "",
+              "lockKey": true
+            },
+            {
+              "label": "<strong>Are you listed as a driver on the insurance for the vehicle you will be using?</strong>",
+              "labelPosition": "top",
+              "labelWidth": "",
+              "labelMargin": "",
+              "optionsLabelPosition": "right",
+              "description": "",
+              "tooltip": "",
+              "customClass": "",
+              "tabindex": "",
+              "inline": false,
+              "hidden": false,
+              "hideLabel": false,
+              "autofocus": false,
+              "disabled": false,
+              "tableView": false,
+              "modalEdit": false,
+              "values": [
+                {
+                  "label": "Yes",
+                  "value": "yes",
+                  "shortcut": ""
+                },
+                {
+                  "label": "No",
+                  "value": "no",
+                  "shortcut": ""
+                }
+              ],
+              "dataType": "",
+              "persistent": true,
+              "protected": false,
+              "dbIndex": false,
+              "encrypted": false,
+              "redrawOn": "",
+              "clearOnHide": true,
+              "customDefaultValue": "",
+              "calculateValue": "",
+              "calculateServer": false,
+              "allowCalculateOverride": false,
+              "validate": {
+                "required": true,
+                "onlyAvailableItems": true,
+                "customMessage": "",
+                "custom": "",
+                "customPrivate": false,
+                "json": "",
+                "strictDateValidation": false,
+                "multiple": false,
+                "unique": false
+              },
+              "errorLabel": "",
+              "errors": "",
+              "key": "additionalTransportListedDriver",
+              "tags": [],
+              "properties": {},
+              "conditional": {
+                "show": "true",
+                "when": "additionalTransportRequested",
+                "eq": "yes",
+                "json": ""
+              },
+              "customConditional": "",
+              "logic": [],
+              "attributes": {
+                "data-cy": "eligibleForAnAdditionalTransportationAllowance"
```

Comment:
Not a blocker and data-cy is not being used anymore. But if used please do not use the old variable name.